src/ducks/notifications/CategoryBudget/utils.js

```diff
@@ -57,3 +57,9 @@ export const buildNotificationView = (client, options) => {
   })
   return notifView
 }
+
+export const hasBudgetAlerts = templateData => {
+  const { budgetAlerts } = templateData
+  const hasMultipleAlerts = budgetAlerts.length > 1
+  return hasMultipleAlerts
+}
```

Comment:
Why do we need to extract this ? I think it makes the code more obscure since you have to go to the utils file to know what's happening.

Reply:
This code is used at several places. I found this more convenient to have it only once.
But I hear the 'makes the code more obscure'.